ksql-serde/src/test/java/io/confluent/ksql/serde/avro/KsqlGenericRowAvroSerializerTest.java

```diff
@@ -391,4 +392,38 @@ public void shouldTransformSourceNameDelimiterForInternal() {
     final GenericRow deserializedKsqlRecord = serde.deserializer().deserialize("topic", bytes);
     assertThat(deserializedKsqlRecord, equalTo(ksqlRecord));
   }
+
+    @Test
+    public void shouldUseSchemaNameFromPropertyIfExists() {
```

Comment:
Can you add a test covering the default case too, i.e. where it uses the current schema name, if there isn't one already.

Reply:
All the tests that use `shouldSerializeTypeCorrectly` (the tests for the different primitive types) already test that the name (and namespace) of the schema match `KsqlConstants.AVRO_SCHEMA_NAMESPACE` by default.